src/index.html

```diff
@@ -20,8 +20,373 @@
       href="styles/index.scss"
```

Comment:
According to the code style rules, SCSS files should be linked as regular CSS files in HTML. Make sure to compile your SCSS into CSS and link the compiled CSS file here.